atd-vzd/migrations/default/1697836965151_add_get_crash_modes_function/up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE OR REPLACE FUNCTION public.units_get_crash_modes (input_crash_id integer)
+	RETURNS json
+	LANGUAGE plpgsql
+	AS $function$
+DECLARE
+	totals json;
+BEGIN
+	SELECT
+		INTO totals json_agg(row_to_json(result))
+	FROM (
+		SELECT
+			vmode.mode_id,
+			mode.atd_mode_category_mode_name AS mode_desc,
+			vmode.unit_id,
+			vmode.death_cnt,
+			vmode.sus_serious_injry_cnt,
+			vmode.nonincap_injry_cnt,
+			vmode.poss_injry_cnt,
+			vmode.non_injry_cnt,
+			vmode.unkn_injry_cnt,
+			vmode.tot_injry_cnt
+		FROM (
+			SELECT
+				unit_id,
+				atd_mode_category AS mode_id,
```

Comment:
This line is updated to use the unit `atd_mode_category` column which is populated by a trigger that runs [alphabetically earlier](https://www.postgresql.org/docs/8.1/triggers.html#:~:text=If%20more%20than%20one%20trigger,input%20to%20the%20next%20trigger.). It used to be a duplicate of the logic in `units_mode_category()` so this should prevent drifts in the future. Checking out `get_crash_modes()` and `atd_txdot_units_create_update()` in the production read replica could help visualize this change.